packages/Material-Core.package/MDColorTheme.class/instance/calculateDarkVariants..st

```diff
@@ -0,0 +1,10 @@
+color-calculating
+calculateDarkVariants: aColor
+
+	| calculatedColors |
+	
+	calculatedColors := (aColor darkShades: self class numberOfShades + 2) asOrderedCollection.
+	calculatedColors remove: calculatedColors first;
```

Comment:
Das ```remove: calculatedColors first;``` sollte eingerückt in die Zeile drunter.

Und warum rechnet ihr da ```+2```? Mir ist nicht ganz klar, wo die herkommt.

Reply:
Ich gehe mal davon aus, dass die hellste und dunkleste Berechnung nicht brauchbar ist, weswegen sie gelöscht werden. Um am Ende aber die gewünschte numberOfShades zu haben, müssen anfangs folglich zwei Farben mehr berechnet werden

Reply:
Robins Vermutung stimmt. Nehmen wir mal die Berechnung der dunklen Varianten: Dann ist das Dunkelste immer Schwarz und das Hellste immer die Farbe von der die Varianten berechnet werden. Die brauchen wir aber nicht wirklich